src/amo/components/EditableCollectionAddon/index.js

```diff
@@ -56,11 +111,59 @@ export class EditableCollectionAddonBase extends React.Component<Props> {
             {i18n.gettext('Remove')}
           </Button>
         </div>
+        {showNotes && (
+          <div className="EditableCollectionAddon-notes">
+            <h4 className="EditableCollectionAddon-notes-header">
+              <Icon name="comments-blue" />
+              {i18n.gettext('User comment')}
+            </h4>
+
+            {this.state.editingNote ? (
+              <React.Fragment>
+                {errorHandler.renderErrorIfPresent()}
+                <DismissibleTextForm
+                  className="EditableCollectionAddon-notes-form"
+                  microButtons
+                  onDelete={this.onDeleteNote}
+                  onDismiss={this.onDismissNoteForm}
+                  onSubmit={this.onSaveNote}
+                  placeholder={i18n.gettext('Add a comment about this add-on.')}
+                  submitButtonText={i18n.gettext('Save')}
+                  text={addon.notes}
+                />
+              </React.Fragment>
+            ) : (
+              <div className="EditableCollectionAddon-notes-read-only">
+                <span
+                  className="EditableCollectionAddon-notes-content"
+                  // eslint-disable-next-line react/no-danger
+                  dangerouslySetInnerHTML={sanitizeHTML(addon.notes)}
+                />
+                <div className="EditableCollectionAddon-notes-buttons">
+                  <Button
+                    buttonType="action"
+                    className="EditableCollectionAddon-notes-edit-button"
+                    micro
+                    onClick={this.onEditNote}
+                  >
+                    {i18n.gettext('Edit')}
+                  </Button>
+                </div>
+              </div>
+            )}
+          </div>
+        )}
       </li>
     );
   }
 }
 
+export const extractId = (ownProps: Props) => {
+  const { addon } = ownProps;
+  return `editable-collection-addon-${addon ? addon.id : ''}`;
```

Comment:
From what I am seeing, the `addon` is always present.

Reply:
True, thanks.